spyder/plugins/console/widgets/internalshell.py

```diff
@@ -129,35 +132,39 @@ def end_input(self, cmd):
 class InternalShell(PythonShellWidget):
     """Shell base widget: link between PythonShellWidget and Interpreter"""
 
-    status = Signal(str)
-    refresh = Signal()
-    go_to_error = Signal(str)
-    focus_changed = Signal()
+    # --- Signals
+
+    # This signal is emitted when the buffer is flushed
+    sig_refreshed = Signal()
+
+    # This signal is emitted to request a sending a status message

```

Comment:
Please change this to

    # Request to show a status message on the main window

Reply:
Fixed